dist/io.github.freedoom.Phase1.desktop

```diff
@@ -1,6 +1,5 @@
 [Desktop Entry]
-Name=Phase 1
-Name[he]=מופע 1
```

Comment:
I doubt we care much about the loss of this Hebrew translation since It's probably fine now that `Phase` is gone anyway.